### DISCUSSION TITLE
Sort translation strings

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17130] [Feature Requests]
It would be nice if translation strings could be ordered alphabetically. At the list view (`/admin/settings/translation-strings`) as well as in the select dropdown.

For the list view i would like to see the same possibilities as we have with other collections: order, search, filter, resize the colums, etc. the items.

I am working a lot with translation strings, when the list grows and grows it gets hard to maintain.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
